ckan/lib/formatters.py

```diff
@@ -137,7 +137,7 @@ def months_between(date1, date2):
         'day': datetime_.day,
         'year': datetime_.year,
         'month': _MONTH_FUNCTIONS[datetime_.month - 1](),
-        'timezone': datetime_.tzinfo.zone,
+        'timezone': datetime_.tzinfo._tzname,
```

Comment:
Not saying we can't use it but this looks like an internal property, is it safe to use?

And BTW what happens if the `datetime` object provided does not have a `tzinfo` available? My understanding is that these are not set by default (eg if creating a date like `datetime(2016, 2, 4)`). (I know this is out of the scope of this PR)